content/css/concepts/filter-functions/terms/blur/blur.md

```diff
@@ -33,3 +33,5 @@ Give the image a blur radius of 4px:
   filter: blur(4px);
 }
 ```
+
+![css-filterfunctions-blur-example](https://github.com/DanWSDev/docs/blob/patch-3/media/css-filterfunctions-blur-example.png)
```

Comment:
the image should be in the docs/media folder and the link should use the following path -
https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunctions-blur-example.png